Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -32,9 +32,8 @@ extension LLMLocalSession {
             return
         }
         
-        let prompt = modelConfiguration.prepare(prompt: formattedChat)
         let promptTokens = await modelContainer.perform { _, tokenizer in
-            tokenizer.encode(text: prompt)
+            tokenizer.encode(text: formattedChat)
```

Comment:
In SpeziLLM's current form, the change is valid as the `modelConfiguration.prepare()` function presently does not perform any operation as the respective prompt preparation closure in the MLX `ModelConfiguration` is not configurable via the init of `LLMLocalSchema`. However, that might change in the future, leading to unintended consequences.

I defer the final decision to @LeonNissen, who is responsible for the majority of the MLX local LLM inference implementation.